beacon_node/beacon_chain/src/errors.rs

```diff
@@ -161,6 +161,7 @@ pub enum BeaconChainError {
     BlockRewardAttestationError,
     BlockRewardSyncError,
     SyncCommitteeRewardsSyncError,
+    AttestationRewardsSyncError,
```

Comment:
Ahh I missed the fact the sync rewards error repeats the word `Sync`, but we shouldn't mention `Sync` here at all

```suggestion
    AttestationRewardsError,
```